spec/tasks/connectors/cylera/cylera_task_spec.rb

```diff
@@ -50,5 +50,23 @@
         expect { task.run(options) }.to raise_error(SystemExit) { |e| expect(e.status).to_not be_zero }
       end
     end
+
+    context 'when time value is unexpected' do
```

Comment:
"when time value is an unexpected format"?

Reply:
Updated